### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by DaanVanHauwermeiren**
A simple cleanup of all the files:
remove some old code, check if the notebooks still run on new julia and pluto version

- update gitignore
- remove legacy code
- update project and manifest: only need pluto as the other libs are loaded automatically in the notebook
- update gitignore: we need the notebooks folder
- update notebooks to new julia version and new libs, all seem ok except for issues 145 146 147

### CODE REVIEW

File: legacy/examples/error_propagation.jl [hunk 1]
```diff
@@ -1,46 +0,0 @@
-#=
-Created on 03/02/2021 09:20:46
```
Comment (Beramos) [inline legacy/examples/error_propagation.jl:2]: Might be nice to keep this one in?

File: notebooks/day1/03-exercises backup 1 backup 1.jl
Comment (Beramos) [inline notebooks/day1/03-exercises backup 1 backup 1.jl:1]: I think we need another backup of this file :rofl: 

On a serious note pls remove.

File: notebooks/day1/03-exercises backup 1.jl
Comment (Beramos) [inline notebooks/day1/03-exercises backup 1.jl:1]: On a serious note pls remove.

File: notebooks/day1/aprettybird.jpg
Comment (Beramos) [inline notebooks/day1/aprettybird.jpg:1]: I remember this img being downloaded from url so no file should be here. I checked and could not find a reference to this file. Please double check though.

Comment (Beramos) [reply notebooks/day1/aprettybird.jpg:1]: https://github.com/search?q=repo%3ABeramos%2FDS-Julia2925%20aprettybird&type=code

File: notebooks/examples/performance_convolution backup 1.jl
Comment (Beramos) [inline notebooks/examples/performance_convolution backup 1.jl:1]: please remove

File: notebooks/examples/performance_sum backup 1.jl
Comment (Beramos) [inline notebooks/examples/performance_sum backup 1.jl:1]: please remove

**Review by Beramos — CHANGES_REQUESTED** [review]
Some rogue files but other than that thanks for the effort!

File: legacy/examples/error_propagation.jl (hunk 1 above)
Comment (Beramos) [reply legacy/examples/error_propagation.jl:2]: With in, I mean move to examples.

**Comment by Beramos** [comment]
I would in general retain a little more examples.

File: notebooks/day1/03-exercises backup 1 backup 1.jl
Comment (DaanVanHauwermeiren) [reply notebooks/day1/03-exercises backup 1 backup 1.jl:1]: oh no, I most likely did a `git add *` :cry:

**Comment by DaanVanHauwermeiren** [comment]
> I would in general retain a little more examples.

going to rewind and move `legacy/examples` to `examples` + add a readme

File: notebooks/day1/01-basics.jl [hunk 2]
```diff
@@ -800,23 +800,48 @@ stirling(n) = missing
...
 md"""
-> **Question 6: time is relative**
+> **Question 6: Narcissistic numbers**
 
-The function `time` returns the current Greenwich Mean Time in seconds since "the epoch", which is an arbitrary time used as a reference point. On UNIX systems, the epoch is 1 January 1970.
-Write a script that reads the current time in seconds (use `time()`) and converts it to a time of day in hours, minutes, and seconds, plus the number of days since the beginning of the year.
+[source](https://www.w3resource.com/python-exercises/basic/)
+
+If you are a reader of Greek mythology, then you are probably familiar with Narcissus. He was a hunter of exceptional beauty, and he died because he was unable to leave a pool after falling in love with his own reflection. That's why I keep myself away from pools these days (kidding).
+In mathematics, he has kins called narcissistic numbers - numbers that can't get enough of themselves. In particular, they are numbers that are the sum of their digits when raised to the power of the number of digits.
```
Comment (Beramos) [inline notebooks/day1/01-basics.jl:808]: typo @MichielStock

Comment (Beramos) [reply notebooks/day1/01-basics.jl:808]: he has kins...  not sure what that means or just typo

**Review by Beramos — APPROVED** [review]
I'm merging this. I will move the typo to separate issue

**Pull request merged**